lib/components.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const path = require('path');
+const log = require('./logger')('nodecg/lib/components');
+const Bundles = require('./bundle-manager');
+
+log.trace('Adding Express routes for bundle components');
+
+const all = Bundles.all();
+
+for (let i = 0; i < all.length; i++) {
+	const bundle = all[i];
+	if (bundle.mount) {
```

Comment:
By inverting this condition to be `if (!bundle.mount || bundle.mount.length <= 0)` (or something), and then `continue;`ing (or `return;`ing if using `Array.forEach`), the code currently inside the `if` block can be moved left by one level of indentation, making it a bit nicer to read. I usually take this stylistic approach whenever I can. For example:

```js
foo.forEach(bar => {
    if (!bar.baz) {
        // Nothing for us to do then.
        return;
    }

    // Do something with baz.
    foo.bar.baz();
});
```